### PR TITLE
doc: clean up limitation section on the main readme page

### PR DESCRIPTION
**Pull request by kalyazin**
## Changes

Re(move) topic-specific issues and limitations from the main README page.
Only one limitation remains there that does not have a better location as of now.

## Reason

We have a number of topic-specific limitations listed on the main README page that cause confusion about where a new limitation needs to be added, to the main README or elsewhere.

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license. For more information on following
Developer Certificate of Origin and signing off your commits, please check
[`CONTRIBUTING.md`][3].

## PR Checklist

- ~[ ] If a specific issue led to this PR, this PR closes the issue.~
- [x] The description of changes is clear and encompassing.
- [x] Any required documentation changes (code and docs) are included in this PR.
- ~[ ] API changes follow the [Runbook for Firecracker API changes][2].~
- ~[ ] User-facing changes are mentioned in `CHANGELOG.md`.~
- ~[ ] All added/changed functionality is tested.~
- ~[ ] New `TODO`s link to an issue.~
- [x] Commits meet [contribution quality standards](https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md#contribution-quality-standards).

---

- [x] This functionality cannot be added in [`rust-vmm`][1].

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md
[3]: https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,13 +140,8 @@ We test all combinations of:
 
 ## Known issues and Limitations
 
-- The [SendCtrlAltDel](docs/api_requests/actions.md#sendctrlaltdel) API request
-  is not supported for aarch64 enabled microVMs.
-- If a CPU template is not used on x86_64, overwrites of `MSR_IA32_TSX_CTRL` MSR
-  value will not be preserved after restoring from a snapshot.
 - The `pl031` RTC device on aarch64 does not support interrupts, so guest
```
Comment (sudanl0) [inline README.md:143]: Is there a way to remove/move the entire section?

Comment (kalyazin) [reply README.md:143]: yes, I was aiming to do that, but couldn't find a place for the ARM/RTC issue. Do you have a suggestion in mind?

File: README.md [hunk 2]
```diff
@@ -140,13 +140,8 @@ We test all combinations of:
 
 ## Known issues and Limitations
 
-- The [SendCtrlAltDel](docs/api_requests/actions.md#sendctrlaltdel) API request
```
Comment (xmarcalx) [inline README.md:143]: where this line ended up?

**Review by pb8o — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pb8o) [reply README.md:143]: Maybe an arm.md or clocks.md? But they would be pretty short at the moment :)

File: README.md (hunk 2 above)
Comment (kalyazin) [reply README.md:143]: this is where it is already described: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api_requests/actions.md#intel-and-amd-only-sendctrlaltdel

File: README.md (hunk 1 above)
Comment (kalyazin) [reply README.md:143]: yeah, doesn't seem to be worth it.

Comment (sudanl0) [reply README.md:143]: We can have a Limitations.md for generic stuff that doesn't fit anywhere else but then the argument would be why there and not here so, I don't have a good suggestion right now. I'm ok with the current change.

**Review by sudanl0 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (xmarcalx) [reply README.md:143]: thanks

**Pull request merged**